src/components/widgets/registration/useGetReviewInfo.tsx

```diff
@@ -93,7 +94,14 @@ export const useGetReviewInfo = (
                     onEditClick={() => onClick(STEP_TYPE.USER_INFO)}
                 >
                     <Typography.Paragraph>
-                        Telegram: {userNameTelegram}
+                        <div>
+                            Telegram:{' '}
```

Comment:
Well)) this is a wrong usage for Typograpy.Paragraph. Just create to tags and make them under Flex. Also, you have conditions, but fields are required right?

Reply:
>Also, you have conditions, but fields are required right?

Not entirely true, these fields are conditional. One contact out of two required